cmd/event_display/main.go

```diff
@@ -85,16 +89,51 @@ func run(ctx context.Context) {
 	if err != nil {
 		log.Fatal("Failed to create client: ", err)
 	}
-	conf, err := config.JSONToTracingConfig(os.Getenv("K_CONFIG_TRACING"))
+
+	cfg := &observability.Config{}
+
+	err = json.Unmarshal([]byte(os.Getenv("K_OBSERVABILITY_CONFIG")), cfg)
+	cfg = observability.MergeWithDefaults(cfg)
+
+	ctx = observability.WithConfig(ctx, cfg)
+
+	otelResource := resource.Default("hearbeat")
+
+	meterProvider, err := metrics.NewMeterProvider(
+		ctx,
+		cfg.Metrics,
+		metric.WithResource(otelResource),
+	)
 	if err != nil {
-		log.Printf("Failed to read tracing config, using the no-op default: %v", err)
+		log.Panicf("failed to setup meter provider: %s", err.Error())
```

Comment:
You're converting this from a "whinge and continue" to "hard down".  Is that intentional?